### PR TITLE
New cloud subpage "Availability & Performance"

### PR DESCRIPTION
**Pull request by sajumb**

### CODE REVIEW

**Comment by sajumb** [comment]
I have made the changes after the vale linter run that I believe it worth fixing.
- I do not believe it is relevant to define "CPU" as all people know what it is.
- I believe the reading experience will be less suitable to split up 27-28 word sentences into two smaller sentences.
But feel free to do this. I am not able to write shorter sentences than what is included in the PR. 😃

**Comment by jonat123** [comment]
@sajumb  thanks for the PR 😄 

I'll review it tomorrow morning and make sure it is merged in tomorrow as well 😄

File: umbraco-cloud/getting-started/umbraco-cloud-plans.md [hunk 1]
```diff
@@ -18,30 +18,28 @@ Currently, all available Umbraco Cloud plans are utilizing P1V3 Azure App Servic
...
-- CPU - 50%
-- RAM - 2000 MB
+- CPU - 50% (210 sec of CPU time for a 5 minute period)
+- Memory - 2000 MB (in private bytes)
 - Disk - 10400 MB
 
-## Quotas for the shared pool
-
-Whenever a pool hits above 95% usage of any resource disk, the busiest staging or development environment, that hasn't been moved previously will be moved to a different pool.
+You will be able to see the current and past CPU and
```
Comment (jonat123) [inline umbraco-cloud/getting-started/umbraco-cloud-plans.md:45]: @sajumb something is missing here 🤔

File: umbraco-cloud/set-up/project-settings/README.md [hunk 2]
```diff
@@ -113,13 +114,9 @@ You can change your Umbraco Cloud project to run in a dedicated setup with addit
 
 On your Umbraco Cloud project, it is possible to see the usage of Custom Domains, Media Storage, Content Nodes, and Bandwidth for the project. You can also check if it is using above or below the allowed amount for the plan that your project is on.
 
-## [Payment](../manage-subscriptions/new-shop.md)
-
-From here, you can see the payment history for your project, change the credit card, as well as download the latest invoices for your project.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/README.md:118]: @sajumb any reason why you are deleting this?

File: umbraco-cloud/getting-started/umbraco-cloud-plans.md (hunk 1 above)
Comment (sajumb) [reply umbraco-cloud/getting-started/umbraco-cloud-plans.md]: Yup, my bad. I have deleted the incomplete sentence and rearranged a bit in a new commit.

File: umbraco-cloud/set-up/project-settings/README.md (hunk 2 above)
Comment (sajumb) [reply umbraco-cloud/set-up/project-settings/README.md:118]: The page has been removed. Now the user will see only be able to see this information through the organization page.

File: umbraco-cloud/set-up/project-settings/README.md [hunk 3]
```diff
@@ -113,13 +114,9 @@ You can change your Umbraco Cloud project to run in a dedicated setup with addit
...
+## [Availability & Performance](availability-performance.md)
 
-{% hint style="info" %}
-This only applies to new customers on Umbraco Cloud after the 28th of April, 2021. For old projects, the invoices and payment history are still available from the [Umbraco Shop](https://shop.umbraco.com/profile/sign-in).
-{% endhint %}
```
[the remark this answers is not in the document]
Comment (sajumb) [reply umbraco-cloud/set-up/project-settings/README.md:122]: The page has been removed. Now the user will see only be able to see this information through the organization page.

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+
+### Issue Discovery
+The feature enables users to discover application issues that may impact the overall performance of their Umbraco Cloud projects. By analyzing Azure metrics data, you gain valuable insights into the performance of your app service, including response times and CPU/memory usage.
+
+Real-time monitoring of HTTPS status codes helps you identify any errors or disruptions to your application's availability. This proactive approach allows you to respond swiftly, ensuring that your website or application remains accessible and responsive to users.
+
+### User-Friendly Interface
+Umbraco Cloud provides a user-friendly interface for accessing and interpreting the "Availability and Performance" feature. The interface presents Azure metrics data in a clear and understandable manner, making it easy for users to navigate and gain actionable insights. With intuitive SVG-based visualizations and highlighted statistics for the selected time range, you can monitor the health and performance of your cloud projects effortlessly.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:101]:
```suggestion
Umbraco Cloud provides a user-friendly interface for accessing and interpreting the "Availability and Performance" feature. The interface presents Azure metrics data in a clear and understandable manner, making it easier for users to navigate and gain actionable insights. With intuitive SVG-based visualizations and highlighted statistics for the selected time range, you can monitor the health and performance of your cloud projects effortlessly.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+For cloud projects with a dedicated option (or a shared plan with another granularity than 5 minutes), the user will see the average assigned private bytes in bytes.
+Here the statistics panel will display the maximum, average, and minimum allocation of private bytes based on selected granularity.
+
+## Key benefits
+Some of the benefits the “Availability & Performance” feature gives you are as follows.
+
+### Health Monitoring
+Umbraco Cloud's "Availability and Performance" feature allows users to monitor the health of their cloud projects more effectively. By leveraging Azure metrics data, you can access critical information such as HTTPS status codes, response times, CPU time, and memory usage in private bytes.
+
+The CPU and memory usage is particularly important to keep an eye on from time to time in case the cloud project is running on a shared plan to ensure that the plan quotas aren’t exceeded.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:93]:
```suggestion
 The CPU and memory usage is particularly important to keep an eye on from time to time. 
 This is to ensure the cloud project running on a shared that the plan quotas aren’t exceeded.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+
+![Memory usage](../images/AP-4-MemoryUsage.png)
+
+For cloud projects using a shared resource and a granularity of 5 minutes, the user will see both the assigned private bytes in mega bytes (MB) and a comparison against the plan quota. The plan quotas are described here<https://docs.umbraco.com/umbraco-cloud/getting-started/umbraco-cloud-plans>. 
+
+For cloud projects with a dedicated option (or a shared plan with another granularity than 5 minutes), the user will see the average assigned private bytes in bytes.
+Here the statistics panel will display the maximum, average, and minimum allocation of private bytes based on selected granularity.
+
+## Key benefits
+Some of the benefits the “Availability & Performance” feature gives you are as follows.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:88]:
```suggestion
Below you can read about the benefits with the “Availability & Performance” feature gives.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 7]
```diff
@@ -0,0 +1,106 @@
...
+Here the statistics panel will display the maximum, average, and minimum CPU time based on selected granularity.
+
+#### Memory Usage
+The chart shows the memory usage in private bytes consumed by the application in the selected time range with time periods equalling the selected granularity.
+
+![Memory usage](../images/AP-4-MemoryUsage.png)
+
+For cloud projects using a shared resource and a granularity of 5 minutes, the user will see both the assigned private bytes in mega bytes (MB) and a comparison against the plan quota. The plan quotas are described here<https://docs.umbraco.com/umbraco-cloud/getting-started/umbraco-cloud-plans>. 
+
+For cloud projects with a dedicated option (or a shared plan with another granularity than 5 minutes), the user will see the average assigned private bytes in bytes.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:84]:
```suggestion
For cloud projects with a dedicated option (or shared plans with another granularity than 5 minutes), users will see the average assigned private bytes in bytes.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 8]
```diff
@@ -0,0 +1,106 @@
...
+
+For cloud projects with a dedicated option (or a shared plan with another granularity than 5 minutes), the user will see the average assigned CPU time in seconds.
+Here the statistics panel will display the maximum, average, and minimum CPU time based on selected granularity.
+
+#### Memory Usage
+The chart shows the memory usage in private bytes consumed by the application in the selected time range with time periods equalling the selected granularity.
+
+![Memory usage](../images/AP-4-MemoryUsage.png)
+
+For cloud projects using a shared resource and a granularity of 5 minutes, the user will see both the assigned private bytes in mega bytes (MB) and a comparison against the plan quota. The plan quotas are described here<https://docs.umbraco.com/umbraco-cloud/getting-started/umbraco-cloud-plans>.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:82]:
```suggestion
Cloud projects on shared resources with a granularity of 5 minutes, will see the assigned private bytes in megabytes (MB) and a comparison against the [plan quota](https://docs.umbraco.com/umbraco-cloud/getting-started/umbraco-cloud-plans).

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 9]
```diff
@@ -0,0 +1,106 @@
...
+#### CPU Usage
+The chart depicts the CPU time consumed by the application in the selected time range with time periods equalling the selected granularity.
+
+![Cpu usage](../images/AP-3-CpuUsage2.png)
+
+For cloud projects using a shared resource and a granularity of 5 minutes, the user will see both the assigned CPU time in seconds and a comparison against the plan quota. The plan quotas are described here<https://docs.umbraco.com/umbraco-cloud/getting-started/umbraco-cloud-plans>. 
+In this case, the statistics panel maximum CPU time, average CPU, plan quota, and the maximum and average percentage of the consumed CPU in a 5 minute period compared to the plan quota.
+
+For cloud projects with a dedicated option (or a shared plan with another granularity than 5 minutes), the user will see the average assigned CPU time in seconds.
+Here the statistics panel will display the maximum, average, and minimum CPU time based on selected granularity.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:75]:
```suggestion
Cloud projects on dedicated options (or a shared plan with another granularity than 5 minutes), users will see the average assigned CPU time in seconds.
Here the statistics panel will display the maximum, average, and minimum CPU time based on selected granularity.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 10]
```diff
@@ -0,0 +1,106 @@
...
+
+The statistics panel shows the average, maximum, and minimum response for the shown data points.
+
+#### CPU Usage
+The chart depicts the CPU time consumed by the application in the selected time range with time periods equalling the selected granularity.
+
+![Cpu usage](../images/AP-3-CpuUsage2.png)
+
+For cloud projects using a shared resource and a granularity of 5 minutes, the user will see both the assigned CPU time in seconds and a comparison against the plan quota. The plan quotas are described here<https://docs.umbraco.com/umbraco-cloud/getting-started/umbraco-cloud-plans>. 
+In this case, the statistics panel maximum CPU time, average CPU, plan quota, and the maximum and average percentage of the consumed CPU in a 5 minute period compared to the plan quota.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:72]:
```suggestion
In this case, the statistics panel shows the following:
- The maximum CPU time 
- The average CPU
- The plan quota
- The maximum and average percentage of the consumed CPU in a 5-minute period compared to the plan quota.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 11]
```diff
@@ -0,0 +1,106 @@
...
+![AppPerformance](../images/AP-2-AppPerformance.png)
+
+The statistics panel shows the average, maximum, and minimum response for the shown data points.
+
+#### CPU Usage
+The chart depicts the CPU time consumed by the application in the selected time range with time periods equalling the selected granularity.
+
+![Cpu usage](../images/AP-3-CpuUsage2.png)
+
+For cloud projects using a shared resource and a granularity of 5 minutes, the user will see both the assigned CPU time in seconds and a comparison against the plan quota. The plan quotas are described here<https://docs.umbraco.com/umbraco-cloud/getting-started/umbraco-cloud-plans>.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:71]:
```suggestion 
Cloud projects using a shared resource and a granularity of 5 minutes, users will see the assigned CPU time in seconds and a comparison against the [plan quota](https://docs.umbraco.com/umbraco-cloud/getting-started/umbraco-cloud-plans).  
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 12]
```diff
@@ -0,0 +1,106 @@
...
+- Failed Requests: when one or more server errors have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 100% of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 100% the plan quota in a 5 minute period during the selected time range.
+
+A warning is shown in case the CPU Usage or Memory Usage
+- Failed Requests: when one or more client errors (but no server errors) have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 80% percent of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 80% percent of the plan quota in a 5 minute period during the selected time range.
+
+Errors and warnings for CPU Usage and Memory Usage are only shown for cloud projects on a shared plan and with a granularity of 5 minutes selected.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:46]:
```suggestion
Errors and warnings for CPU Usage and Memory Usage are only shown for cloud projects on shared plans with a granularity of 5 minutes selected.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 13]
```diff
@@ -0,0 +1,106 @@
...
+
+An error indicator is shown in the following situations:
+- Failed Requests: when one or more server errors have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 100% of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 100% the plan quota in a 5 minute period during the selected time range.
+
+A warning is shown in case the CPU Usage or Memory Usage
+- Failed Requests: when one or more client errors (but no server errors) have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 80% percent of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 80% percent of the plan quota in a 5 minute period during the selected time range.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:44]:
```suggestion
- **Failed Requests**: when one or more client errors (but no server errors) have occurred in the selected time range.
- **CPU Usage**: when the maximum CPU time has exceeded 80% percent of the plan quota in a 5-minute period during the selected time range.
- **Memory Usage**: when the maximum private time has exceeded 80% percent of the plan quota in a 5-minute period during the selected time range.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 14]
```diff
@@ -0,0 +1,106 @@
...
+![Pane-view](../images/AP-Panel-Selector.png)
+
+Each tile includes relevant statistics and potentially a warning or an error indicator in case there is something you might want to consider.
+
+An error indicator is shown in the following situations:
+- Failed Requests: when one or more server errors have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 100% of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 100% the plan quota in a 5 minute period during the selected time range.
+
+A warning is shown in case the CPU Usage or Memory Usage
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:41]: @sajumb I feel like something is missing from this sentence. 🤔

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 15]
```diff
@@ -0,0 +1,106 @@
...
+The panel selector consists of four tiles, each representing a specific segment of data. The four segments are failed request, App Performance, CPU Usage, and Memory Usage.
+
+![Pane-view](../images/AP-Panel-Selector.png)
+
+Each tile includes relevant statistics and potentially a warning or an error indicator in case there is something you might want to consider.
+
+An error indicator is shown in the following situations:
+- Failed Requests: when one or more server errors have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 100% of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 100% the plan quota in a 5 minute period during the selected time range.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:39]:
```suggestion
An error indicator is shown in the following situations:
- **Failed Requests**: when one or more server errors have occurred in the selected time range.
- **CPU Usage**: when the maximum CPU time has exceeded 100% of the plan quota in a 5-minute period during the selected time range.
- **Memory Usage**: when the maximum private time has exceeded 100% of the plan quota in a 5-minute period during the selected time range.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 16]
```diff
@@ -0,0 +1,106 @@
...
+
+When you enter the page, you will a default visualization of the failed requests for the last 24 hours with a data point set for every fifth minute. You are able to change the time range to a predefined interval or define a specific start and end time. You can also select the granularity of the data points.
+
+![Time-range-selector](../images/AP-time-range-selector.png)
+
+Initially, you will only be able to set the time granularity to “5 minutes”.
+
+### Panel view
+
+The panel selector consists of four tiles, each representing a specific segment of data. The four segments are failed request, App Performance, CPU Usage, and Memory Usage.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:30]:
```suggestion
The panel selector consists of four tiles, each representing a specific segment of data. The four segments are **failed request**, **App Performance**, **CPU Usage**, and **Memory Usage**.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 17]
```diff
@@ -0,0 +1,106 @@
...
+- Panel view
+- Chart and statistics view
+
+![AP-Overview](../images/AP-Overview.png)
+
+Note: The page initially display metrics for the live environment for all Umbraco Cloud projects. More detailed visualization and tools intended for troubleshooting to be added in the future will be restricted to “Standard” and “Professional” project plans.
+
+### Time range and granularity selector
+
+When you enter the page, you will a default visualization of the failed requests for the last 24 hours with a data point set for every fifth minute. You are able to change the time range to a predefined interval or define a specific start and end time. You can also select the granularity of the data points.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:22]:
```suggestion
When entering the page, you'll see a default visualization of failed requests for the last 24 hours with a data point set for every fifth minute. You are able to change the time range to a predefined interval or define a specific start and end time. You can also select the granularity of the data points.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 18]
```diff
@@ -0,0 +1,106 @@
...
+Under Settings in the Umbraco Cloud Portal, you'll find Availability & Performance. This is where you find the page that shows visualization and statistics for four sections.
+
+The page consists of the following parts:
+- Time range and granularity selector
+- Panel view
+- Chart and statistics view
+
+![AP-Overview](../images/AP-Overview.png)
+
+Note: The page initially display metrics for the live environment for all Umbraco Cloud projects. More detailed visualization and tools intended for troubleshooting to be added in the future will be restricted to “Standard” and “Professional” project plans.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:18]:
```suggestion
{% hint style="info" %}
The page initially displays metrics for the live environment for all Umbraco Cloud projects. More detailed visualization and tools intended for troubleshooting to be added in the future will be restricted to “Standard” and “Professional” project plans.
{% endhint %}
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 19]
```diff
@@ -0,0 +1,106 @@
...
+
+In the Umbraco Cloud Settings menu you can find a page called _Availability & Performance_.
+
+Umbraco Cloud's "Availability & Performance" feature lets you see get an overview of your cloud projects' past and current health. Leveraging Azure metrics data, the page provides users with valuable insights into the availability and performance of the cloud project enabling them to monitor and address any issues that may impact the user experience.
+
+## Overview
+
+Under Settings in the Umbraco Cloud Portal, you'll find Availability & Performance. This is where you find the page that shows visualization and statistics for four sections.
+
+The page consists of the following parts:
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:11]:
```suggestion
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 20]
```diff
@@ -0,0 +1,106 @@
+# Availability & Performance
+
+In the Umbraco Cloud Settings menu you can find a page called _Availability & Performance_.
+
+Umbraco Cloud's "Availability & Performance" feature lets you see get an overview of your cloud projects' past and current health. Leveraging Azure metrics data, the page provides users with valuable insights into the availability and performance of the cloud project enabling them to monitor and address any issues that may impact the user experience.
+
+## Overview
+
+Under Settings in the Umbraco Cloud Portal, you'll find Availability & Performance. This is where you find the page that shows visualization and statistics for four sections.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:9]:
```suggestion
Under Availability & Performance, you'll find visualization and statistics for four sections:
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 21]
```diff
@@ -0,0 +1,106 @@
+# Availability & Performance
+
+In the Umbraco Cloud Settings menu you can find a page called _Availability & Performance_.
+
+Umbraco Cloud's "Availability & Performance" feature lets you see get an overview of your cloud projects' past and current health. Leveraging Azure metrics data, the page provides users with valuable insights into the availability and performance of the cloud project enabling them to monitor and address any issues that may impact the user experience.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:5]:
```suggestion
The "Availability & Performance" feature lets you see get an overview of your cloud projects' past and current health. Leveraging Azure metrics data, the page provides users with valuable insights into the availability and performance of the cloud project enabling them to monitor and address any issues that may impact the user experience.
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 22]
```diff
@@ -0,0 +1,106 @@
+# Availability & Performance
+
+In the Umbraco Cloud Settings menu you can find a page called _Availability & Performance_.
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:3]:
```suggestion

```

File: umbraco-cloud/set-up/project-settings/README.md (hunk 2 above)
Comment (jonat123) [reply umbraco-cloud/set-up/project-settings/README.md:118]: Is that also the case for users in the old shop, or is it just for users in the new one?

**Review by jonat123 — CHANGES_REQUESTED** [review]
@sajumb Looks good overall, I noticed some links not formated properly and so on.

I also tried to shorten some of the sentences that was to long :) 

Please have a look at my suggestions :D

**Review by jonat123 — CHANGES_REQUESTED** [review]
@sajumb Looks good overall, I noticed some links not formated properly and so on.

I also tried to shorten some of the sentences that was to long :) 

Please have a look at my suggestions :D

**Review by jonat123 — CHANGES_REQUESTED** [review]
@sajumb Looks good overall, I noticed some links not formated properly and so on.

I also tried to shorten some of the sentences that was to long :) 

Please have a look at my suggestions :D

File: umbraco-cloud/set-up/project-settings/README.md (hunk 2 above)
Comment (sajumb) [reply umbraco-cloud/set-up/project-settings/README.md:118]: Good point. I might have been too quick as I am not aware of this 😕  I will consult KEC and come back to you on this one.

Comment (jonat123) [reply umbraco-cloud/set-up/project-settings/README.md:118]: Great! We also did updated the article for payments in the new shop to show that it is paid through the organisation, so I think we should keep them for now, and then we can have a look at moving the article for the new shop to the organisation section instead :)

File: umbraco-cloud/set-up/project-settings/availability-performance.md (hunk 10 above)
Comment (sajumb) [reply umbraco-cloud/set-up/project-settings/availability-performance.md:72]: Yup, nice improvement. The second line "The average CPU" should include " time" as well though.

File: umbraco-cloud/set-up/project-settings/README.md (hunk 2 above)
Comment (sajumb) [reply umbraco-cloud/set-up/project-settings/README.md:118]: KEC states that we should remove the reference to the payment stuff on the project level. That should not be a thing anymore, as we do collective billing and not billing per project. Also for projects in the old shop.

Comment (jonat123) [reply umbraco-cloud/set-up/project-settings/README.md:118]: Okay, great :) 

Thanks for clarifying, then it's fine that we remove it :) 

Make sure to also remove it from the SUMMARY.md file :)

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 23]
```diff
@@ -0,0 +1,107 @@
+# Availability & Performance
+
+
+The "Availability & Performance" feature lets you see get an overview of your cloud projects' past and current health. Leveraging Azure metrics data, the page provides users with valuable insights into the availability and performance of the cloud project enabling them to monitor and address any issues that may impact the user experience.
+
+## Overview
+
+Under Availability & Performance, you'll find visualization and statistics for four sections:
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:8]: @sajumb There is only listed three sections below, is it then correct to say four? 🤔

File: umbraco-cloud/set-up/project-settings/availability-performance.md (hunk 14 above)
Comment (sajumb) [reply umbraco-cloud/set-up/project-settings/availability-performance.md:41]: Yes, it should be: "A warning indicator is shown in the following situations:"

Can you change it?

File: umbraco-cloud/set-up/project-settings/availability-performance.md [hunk 24]
```diff
@@ -0,0 +1,107 @@
...
+![Pane-view](../images/AP-Panel-Selector.png)
+
+Each tile includes relevant statistics and potentially a warning or an error indicator in case there is something you might want to consider.
+
+An error indicator is shown in the following situations:
+- Failed Requests: when one or more server errors have occurred in the selected time range.
+- CPU Usage: when the maximum CPU time has exceeded 100% of the plan quota in a 5 minute period during the selected time range.
+- Memory Usage: when the maximum private time has exceeded 100% the plan quota in a 5 minute period during the selected time range.
+
+A warning is shown in case the CPU Usage or Memory Usage
```
Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:41]:
```suggestion
A warning indicator is shown in the following situations:
```

File: umbraco-cloud/set-up/project-settings/availability-performance.md (hunk 10 above)
Comment (jonat123) [reply umbraco-cloud/set-up/project-settings/availability-performance.md:72]: great I added it to the suggestion :)

File: umbraco-cloud/set-up/project-settings/availability-performance.md (hunk 14 above)
Comment (jonat123) [reply umbraco-cloud/set-up/project-settings/availability-performance.md]: @sajumb Yes, i have changed it :D

File: umbraco-cloud/set-up/project-settings/availability-performance.md (hunk 23 above)
Comment (sajumb) [reply umbraco-cloud/set-up/project-settings/availability-performance.md:8]: 2Yeah, it was changed from four to three sections. It should state "three" here.

Comment (jonat123) [inline umbraco-cloud/set-up/project-settings/availability-performance.md:8]:
```suggestion
Under Availability & Performance, you'll find visualization and statistics for three sections:
```

**Comment by jonat123** [comment]
@sajumb I've just merged in the last suggestions that I made for the PR,
let me know if something looks off :) 

The only thing missing is removing the payment sections from the SUMMARY.MD file, let me know once that is done, then I can get this merged in :)

**Comment by sajumb** [comment]
Hi Jonathan,
Please merge it now. :-)
I will make the change to the payment section in SUMMARY.MD in a later commit. It can definitely wait to Tuesday.

**Pull request merged**